### PR TITLE
Initial commit of template.

### LINKED ISSUE #4: Create flask skeleton project
What the title says...

EXPECTED OUTCOME
A default project template in place on GitHub

### PR DESCRIPTION
**Pull request by fritogotlayed**
### Story
Fixes #4 

### Summary 
Built an initial template for the RAG report project

### Testing
* Navigate to http://127.0.0.1:8000 and verify a 404 page is displayed
* Navigate to http://127.0.0.1:8000/users/me redirects you to login page

### CODE REVIEW

**Comment by fritogotlayed** [comment]
@jbarganier @naitp @Samsanchezz @MrsDbones This is ready for review.

File: app/static/css/bootstrap-theme.css [hunk 1]
```diff
@@ -0,0 +1,587 @@
+/*!
+ * Bootstrap v3.3.5 (http://getbootstrap.com)
+ * Copyright 2011-2015 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ */
+.btn-default,
```
Comment (fritogotlayed) [inline app/static/css/bootstrap-theme.css]: Test Comment

Comment (jbarganier) [inline app/static/css/bootstrap-theme.css]: @fritogotlayed :+1:

**Comment by fritogotlayed** [comment]
This Pull Request is blocking other work.

**Comment by naitp** [comment]
:snail: I'm getting to it.

File: app/users/views.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+@mod.route('/register/', methods=['GET', 'POST'])
+def register():
+    """
+    Registration form
+    """
+    form = RegisterForm(request.form)
+    if form.validate_on_submit():
+        # Create a user instance
+        user = User(anme=form.name.data, email=form.email.data, password=generate_password_hash(form.password.data))
```
Comment (naitp) [inline app/users/views.py]: First parameter of User is misspelled as anme, change it to name.

File: app/users/models.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+    def __init__(self, name=None, email=None, password=None):
+        self.name = name
+        self.email = email
+        self.password = password
+
+    def getStatus(self):
+        return USER.STATUS[self.status]
+
+    def getRole(selfself):
```
Comment (naitp) [inline app/users/models.py]: selfself here should be self, next line breaking due to parameter typo.

**Comment by fritogotlayed** [comment]
@naitp Addressed some of the peer review feedback.1

File: app/users/views.py [hunk 4]
```diff
@@ -0,0 +1,65 @@
+from flask import Blueprint, request, render_template, flash, g, session, redirect, url_for
+from werkzeug import check_password_hash, generate_password_hash
```
Comment (naitp) [inline app/users/views.py]: My editor is reporting "unresolved reference 'check_password_hash'" and "unresolved reference 'generate_password_hash'"

Comment (fritogotlayed) [inline app/users/views.py]: @naitp I have no idea why it's doing that b/c those methods do resolve at run time.

Comment (fritogotlayed) [inline app/users/views.py]: http://werkzeug.pocoo.org/docs/0.10/utils/ is the documentation. It does indeed exist. I assume there's something going on for security reasons.

**Comment by naitp** [comment]
:beer: I've reviewed and commented.  Is good for me, is it good for you?

**Pull request merged**